### PR TITLE
bugfix-62

### LINKED ISSUE #62: Daily numbers in server index are fixed to 12

### PR DESCRIPTION
**Pull request by astar26**
Resolves #62

### CODE REVIEW

File: server/views.py [hunk 1]
```diff
@@ -397,7 +408,8 @@ def export_users_xls(request):
     # Sheet body, remaining rows
     font_style = xlwt.XFStyle()
 
-    rows = Volunteer.objects.all().values_list('full_name', 'first_name', 'last_name', 'tz_number', 'volunteer_type', 'age',
+    rows = Volunteer.objects.all().values_list('full_name', 'first_name', 'last_name', 'tz_number', 'volunteer_type',
+                                               'age',
                                                'phone_number', 'areas', 'city', 'email', 'available_saturday',
                                                'guiding', 'notes')
```
Comment (itay-sho) [inline server/views.py]: but why? just use use list(Volunterr.objects.all())

File: server/views.py [hunk 2]
```diff
@@ -35,16 +35,26 @@ def get_mandatory_areas(request):
...
-            "daily_solved": 12
+            "daily_volunteers": Volunteer.objects.filter(
+                Q(created_date__range=(last_7am,
+                                       next_7am))).count(),  # Tomorrow
+            "daily_help": HelpRequest.objects.filter(
+                Q(created_date__range=(last_7am,
+                                       next_7am))).count(),  # Tomorrow
+            "daily_solved": HelpRequest.objects.filter(Q(created_date__range=(last_7am,
+                                                                              next_7am)),
+                                                       status="DONE").count()
```
Comment (itay-sho) [inline server/views.py]: The readability here is very limited,
can you explain what are you doing here?

regardless think of a way to make this code readable

File: server/views.py [hunk 3]
```diff
@@ -35,16 +35,26 @@ def get_mandatory_areas(request):
 
 @login_required
 def index(request):
+    now = datetime.datetime.now()
+
+    last_7am = datetime.datetime(now.year, now.month, now.day - (1 if now.hour < 7 else 0), 7, 0, 0)
+    next_7am = last_7am + datetime.timedelta(days=1)
     context = {
```
Comment (itay-sho) [inline server/views.py]: just fixing some spacing
```suggestion
    now = datetime.datetime.now()
    last_7am = datetime.datetime(now.year, now.month, now.day - (1 if now.hour < 7 else 0), 7, 0, 0)
    next_7am = last_7am + datetime.timedelta(days=1)
    
    context = {
```

File: server/views.py (hunk 1 above)
Comment (astar26) [reply server/views.py]: Is not really part of my changes, this line changed because of pycharm auto PEP8.

File: server/views.py (hunk 2 above)
Comment (astar26) [reply server/views.py]: Improved, also added comment.

**Pull request merged**